lib/resty/worker/events.lua

```diff
@@ -234,23 +244,25 @@ end
 -- the newly posted event.
 -- @param source string identifying the event source
 -- @param event string identifying the event name
--- @param data the data for the event, anything as long as it can be used with cjson
--- @param unique a unique identifier for this event, providing it will make only 1
+-- @param data the data for the event, anything as long as it can be used
+--   with cjson
```

Comment:
can we line this up similar to https://github.com/Kong/lua-resty-worker-events/pull/21/files#diff-82870c9e68738eefbcd877618ff11b4bR96 ? so no extra indent?

Also for the other occurences of this please

Reply:
fixed